src/application/common/commands-common.ts

```diff
@@ -8,6 +8,7 @@
 import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
+let Globalize = require("globalize");
```

Comment:
why not `import Globalize from 'globalize'`? `import` is preferable to `require`